demo-pipeline/terraform/iam.tf

```diff
@@ -0,0 +1,300 @@
+data "aws_iam_policy_document" "codebuild_trust" {
+  statement {
+    sid     = "CodeBuildAssumeRole"
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["codebuild.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "codepipeline_trust" {
+  statement {
+    sid     = "CodePipelineAssumeRole"
+    effect  = "Allow"
+    actions = ["sts:AssumeRole"]
+
+    principals {
+      type        = "Service"
+      identifiers = ["codepipeline.amazonaws.com"]
+    }
+  }
+}
+
+data "aws_iam_policy_document" "codebuild_service" {
+  statement {
+    sid    = "CloudWatchLogsAccess"
+    effect = "Allow"
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+    resources = ["*"]
+  }
+
+  statement {
+    sid       = "DBSecretsManagerActions"
+    effect    = "Allow"
+    actions   = ["secretsmanager:Get*"]
+    resources = [aws_secretsmanager_secret.db.arn]
+  }
+
+  statement {
+    sid    = "CodePipelineS3BucketAccess"
+    effect = "Allow"
+    actions = [
+      "s3:PutObject",
+      "s3:GetObject",
+      "s3:GetObjectVersion",
+      "s3:GetBucketAcl",
+      "s3:GetBucketLocation"
+    ]
+    resources = [
+      "arn:aws:s3:::codepipeline-us-east-2-*"
```

Comment:
It appears this assumes `us-east-2`; that should be mentioned in the `README`

Reply:
Fixing to use a variable